### DISCUSSION TITLE
當人類跟ai談戀愛？！

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#734] [General]
2013年上映的好萊塢電影《雲端情人》（Her）以中年男子和女AI機器人談戀愛的鋪陳劇情，讓人類對於和AI進行「情感」互動，產生無限的想像空間。電影中的劇情高潮迭起、唯美浪漫，然而與ai談戀愛在現實中真的可行嗎？

在人與人互動的過程中，需要的是「互動」，這包括「陪伴」和「傾聽」，但也涵蓋「爭執」，也就是說，即使透過「大數據」及「AI」技術的結合，可以製造出「完美的理想情人」，但人的「溫度」是AI無法克服的障礙。

我認爲我們人類與ai最大的不同在於我們有「情緒共感」的能力，ai可能在生活技術面上很佔優勢，但他們很難做到「有溫度的陪伴」這件事。而人與人相處有趣的地方在，我們賦予每個話題、每個對話的情緒感知和溫度都不同，這才是互動的意義所在，也是ai無法帶來的快樂。所以身為人類，我想我們更應該學習如何有溫度的好好去表達愛。
共勉之！
https://ct.org.tw/html/news/3-3.php?cat=12&article=1339676

**Comment by Valentinaa0214** [comment]
認同你在AI很難做到「有溫度的陪伴」這件事，不知道為甚麼，與AI談戀愛感覺會很抽離，可能是人類在情緒交流時的共感能力是AI所沒有的嗎？但說不定隨著技術成熟AI或許在未來真的能消除這條與人類明顯的界線感

**Comment by HsuChen88** [comment]
除了肢體接觸時能感受到實際的溫度，日常生活中所有暖心的行為，絕大部分AI應該也能完成吧。
如果真的有一天女生寧願跟AI談戀愛，那我就要去旁邊洗碗。

**Comment by pigeon9975** [comment]
這讓我想到之前有一篇國文課文，內容是人類請ai幫他找理想情人，最後找到的女生卻不是主人翁的，而是他手下的ai的。一想到未來這種事將不再只是科幻小說，而是有可能真實呈現在眼前，就覺得頗驚悚的
